CHANGELOG_PENDING.md

```diff
@@ -5,25 +5,26 @@ Special thanks to external contributors with PRs included in this release:
 BREAKING CHANGES:
 
 * CLI/RPC/Config
+  * [rpc] [\#2391](https://github.com/tendermint/tendermint/issues/2391) /status `result.node_info.other` became a map
 
 * Apps
-  [rpc] /status `result.node_info.other` became a map #[2391](https://github.com/tendermint/tendermint/issues/2391)
+  * [mempool] \#2310 Mempool tracks the `ResponseCheckTx.GasWanted` and enforces `ConsensusParams.BlockSize.MaxGas` on proposals.
 
 * Go API
-  * \#2310 Mempool.ReapMaxBytes -> Mempool.ReapMaxBytesMaxGas
-  * \#2431 Remove Word256 code in libs/common, due to lack of use
+  * [libs/common] \#2431 Remove Word256 code in libs/common, due to lack of use
+
 * Blockchain Protocol
 
 * P2P Protocol
 
 
 FEATURES:
-  * \#2310 Mempool is now aware of the MaxGas requirement
 
 IMPROVEMENTS:
 - [mempool] [\#2399](https://github.com/tendermint/tendermint/issues/2399) Make mempool cache a proper LRU (@bradyjoestar)
-- [types] add Address to GenesisValidator [\#1714](https://github.com/tendermint/tendermint/issues/1714)
+- [types] [\#1714](https://github.com/tendermint/tendermint/issues/1714) Add Address to GenesisValidator
 - [metrics] `consensus.block_interval_metrics` is now gauge, not histogram (you will be able to see spikes, if any)
```

Comment:
is this actually a breaking change? technically the metrics are an API exposed to other processes and hence need to be properly covered by SemVer unless we add an exemption in the readme

Reply:
> is this actually a breaking change?

don't think so. prometheus won't crash because metric's type got changed

Reply:
will the code of any clients need to change ?